react-pool/README.md

```diff
@@ -4,18 +4,16 @@ _Maybe you will find helpful tips_
 
 ## How to use test-net or main-net environments 📋
```

Comment:
This is still misleading and we can adjust later, it is not test-net vs main-net, but remote pool vs local pool.